app/src/main/res/layout/fragment_register_historical_visits.xml

```diff
@@ -75,7 +71,7 @@
                 android:backgroundTint="@color/colorPrimary"
                 android:text="@string/participant_administered_vaccines_action_save"
                 android:textAppearance="@style/ButtonTextStyling"
-                android:visibility="@{viewModel.isEdit ? View.INVISIBLE : View.VISIBLE}"/>
+                android:visibility="@{viewModel.isEdit ? View.INVISIBLE : View.VISIBLE}" />
```

Comment:
You added new space btn visible, enable not to add un neccessary codes that is not needed.